client/src/deepscanToken.ts

```diff
@@ -0,0 +1,122 @@
+      return false;
+   }
+    return true;
+  }
+
+  setToken(token: string) {
+    this._secretStorage.store(this.tokenName, token);
+  }
+
+  async getToken() {
+    return await this._secretStorage.get(this.tokenName);
+  }
+
+  deleteToken() {
+    this._secretStorage.delete(this.tokenName);
+  }
+
+  private async _handleSecretChange(e: vscode.SecretStorageChangeEvent) {
+    vscode.commands.executeCommand('deepscan.updateToken');
+  }
+
+  private _registerCommands() {
+    vscode.commands.registerCommand('deepscan.setToken', async () => {
+      let tokenInput: string = await vscode.window.showInputBox({
+          title: "Configure DeepScan Access Token For VS Code"
+      }) ?? '';
+      tokenInput = tokenInput.trim();
+      if (this._validateToken(tokenInput)) {
+        this.setToken(tokenInput);
+        vscode.window.showInformationMessage(`DeepScan access token has successfully configured.`);
+      }
+    });
+    vscode.commands.registerCommand('deepscan.deleteToken', async () => {
+      const token = await this.getToken();
+      if (token) {
+        const deleteAction = 'Delete';
+        const cancel = 'Cancel';
+        const message = `Are you sure you want to delete your DeepScan access token? The DeepScan extension will no longer be able to inspect.`;
+        const selected = await vscode.window.showWarningMessage(message, deleteAction, cancel);
+        if (selected === deleteAction) {
+          this.deleteToken();
+          vscode.window.showInformationMessage(`Deepscan access token is successfully deleted.`);
+        }
+      } else {
+        vscode.window.showInformationMessage(`Sorry, DeepScan access token doesn't exist.`);
+      }
+    });
+  }
+
+  async showOneOffTokenNotification() {
+    const token = await this.getToken();
+    if (token) {
+        return;
+    }
+
+    const generate = 'Generate';
+    const neverShowAgain = 'Don\'t show again';
+    const message = `DeepScan access token is required for using the DeepScan extension.
+    The token was introduced as a step towards providing personalization features.
+    You can generate a new token by clicking on the button below to go to DeepScan Account Settings page.`;
+    const selected = await vscode.window.showWarningMessage(message, generate, neverShowAgain);
+    if (selected === generate) {
+        vscode.env.openExternal(vscode.Uri.parse(this.tokenRenerateUrl));
+    }
+    return selected;
+  }
+
+  async showEmptyTokenNotification() {
+    const generate = 'Generate';
+    const message = `Sorry, DeepScan access token is not configured.
+    You can generate a new token by clicking on the button below to go to DeepScan Account Settings page.`;
+    const selected = await vscode.window.showErrorMessage(message, generate);
+    if (selected === generate) {
+      vscode.env.openExternal(vscode.Uri.parse(this.tokenRenerateUrl));
+    }
+  }
+
+  async showExpiredTokenNotification() {
+    const regenerate = 'Regenerate';
```

Comment:
버튼이 하나여서 썰렁하기도 하니 다음과 같이 수정하기로 하죠.

```suggestion
    const regenerate = 'Regenerate Token';
```